iModelCore/iModelPlatform/PublicAPI/DgnPlatform/DgnDbTables.h

```diff
@@ -581,6 +581,11 @@ struct EcefLocation
     // Constructor to compute ECEF location from an cartographic origin
     DGNPLATFORM_EXPORT EcefLocation(GeoPointCR originLL, DPoint3dCR originIn, GeoCoordinates::BaseGCSCP GCS);
     DGNPLATFORM_EXPORT Transform ComputeTransform();
+    //! Convert a geographic point to an ECEF (Earth-Centered, Earth-Fixed) location.

```

Comment:
should we cover this with a small unit test ?

Reply:
Yeah, as Paul suggested on Friday --- working on it.

Reply:
pushed